rmath-julia.rb

```diff
@@ -0,0 +1,21 @@
+class RmathJulia < Formula
+  desc "The standalone Rmath library patched to use the dSFMT RNG"
+  homepage "https://github.com/JuliaLang/Rmath-julia"
+  url "https://github.com/JuliaLang/Rmath-julia/archive/v0.1.tar.gz"
+  version "0.1"
+  sha256 "8ba59378273dea999f956cecf37d47723682f3a46b55ce76ec36a0ffdbe6c8f7"
+  head "https://github.com/JuliaLang/Rmath-julia.git", :branch => "master"
+
+  option "with-clang", "Compile using Clang instead of GCC"
+
+  depends_on :gcc => :build unless "with-clang"
```

Comment:
For some reason I saw this and thought "Oh yeah, we do prefer GCC on OSX don't we", until I remembered that no, in fact, it's actually quite difficult to build Julia with GCC on OSX.  @ararslan feel free to test out the `rmath-julia` that is now live in this tap and make sure it works for you, but I am curious as to why you chose this compiler configuration.  Note that on OSX, [`USECLANG` is the default](https://github.com/JuliaLang/julia/blob/9b8e9d45e2264610db6cd057fc4747ae8a6b6249/Make.inc#L358-L359) on everything above Snow Leopard.